lib/screens/onboarding_screen.dart

```diff
@@ -11,66 +11,94 @@ class _OnboardingState extends State<Onboarding> {
+          fit: BoxFit.fitWidth,
+        ),
+        title: "Multiple Categories",
         body:
-            'Eureka provides you with every tool to conveniently collaborate on any problem \n\n Choose from Text chat, Video chat, or Live Video Call ',
-        decoration: const PageDecoration(
-            //pageColor: Color(0xfffdfafc),
-            ),
+            'EureQa allows users to get quick answers to questions of different categories: \n\nAcademic, Lifestyle, Household and Technology.',
+        decoration: getPageDecoration(),
       ),
-      //-----------------------------------------------------------Privacy
       PageViewModel(
-        image: Image.asset('assets/images/privacy.png'),
-        title: 'Your Privacy Matters!',
+        title: 'Collaboration Platform',
         body:
-            'All communication happens in the app, so you don`t have to worry about giving out your personal information to strangers',
-        decoration: const PageDecoration(
-            //pageColor: Color(0xfffdfafc),
-            ),
+            'EureQa provides you with every tool to conveniently collaborate on any problem. \n\n Choose from chat messaging or live video calls.',
+        image: Image.asset(
+          'assets/images/collaboration.png',
+          fit: BoxFit.fitWidth,
+        ),
+        decoration: getPageDecoration(),
       ),
-      //-----------------------------------------------------------Categories
       PageViewModel(
-        image: Image.asset('assets/images/onboarding1.gif'),
-        title: "Multiple categories",
+        title: 'Privacy Matters',
         body:
-            'Eureka has several most popular categories for questions and answers \n\n Lifestyle  Education  Electronics  Household',
-        decoration: const PageDecoration(
-          pageColor: Color(0xfffdfafc),
+            "All communication happens in the app, there's no ndeed to worry about giving out your personal information to strangers.",
+        image: Image.asset(
+          'assets/images/privacy.png',
+          fit: BoxFit.fitWidth,
         ),
+        decoration: getPageDecoration(),
       ),
-      //-----------------------------------------------------------All set
       PageViewModel(
-        image: Image.asset('assets/images/onboarding3.gif'),
-        title: 'You are all set!',
+        title: 'Incentives',
         body:
-            'Asking and answering questions has never been so easy and rewarding \n You are ready to join! \n\n If you still have questions, visit our FAQ section',
-        decoration: const PageDecoration(
-          pageColor: Color(0xff6DB8FF),
+            'EureQa gives you an option of posting questions with an incentive.\n'
+            'Need an answer ASAP? \n\nTry adding incentives to your question. Paid questions get answered a lot faster.',
+        image: Image.asset(
+          'assets/images/incentives.png',
+          fit: BoxFit.fitWidth,
         ),
```

Comment:
This slide is not necessary at this point as paying system has not been integrated yet. Consider comment it out for now